dist/.magento.env.yaml

```diff
@@ -320,6 +320,20 @@
 #                  - consumer1                                                                                        #
 #                  - consumer2                                                                                        #
 #######################################################################################################################
+# LOCK_PROVIDER - use to set the lock provider.                                                                       #
```

Comment:
I recommend reminding them what it does. Maybe something like:

"The lock provider prevents the launch of duplicate cron jobs and cron groups. You must use the `file` lock provider in the Production environment. Starter environments and the Pro Integration environment do not use the MAGENTO_CLOUD_LOCKS_DIR variable, so ece-tools applies the `db` lock provider automatically."